res/css/views/rooms/_EventBubbleTile.scss

```diff
@@ -708,20 +722,6 @@ limitations under the License.
     }
 }
 
-/* events that do not require bubble layout */
-.mx_GenericEventListSummary[data-layout=bubble],
-.mx_EventTile.mx_EventTile_bad[data-layout=bubble] {
```

Comment:
`.mx_EventTile.mx_EventTile_bad[data-layout=bubble]` actually seems to require the bubble layout. Removing the line requires `.mx_GenericEventListSummary[data-layout=bubble]` to be merged to the block above.